.github/workflows/docker_linux.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           tags: ${{ env.IMAGE_TAG }}
 
       - name: Push Linux image
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
```

Comment:
still not clear why changes in github workflows are needed

Reply:
It was my mistake. I shouldn't add this file